app/services/account_reset/delete_account.rb

```diff
@@ -39,6 +39,17 @@ def track_mfa_method_counts
       @mfa_method_counts = MfaContext.new(user).enabled_two_factor_configuration_counts_hash
     end
 
+    def verification_method
+      return nil if !user.identity_verified?
+      if user.in_person_enrollments.present?
+        :in_person_proofing
+      elsif user.active_profile.gpo_confirmation_codes.present?
+        :verify_by_mail
+      elsif user.identity_verified_with_biometric_comparison?
+        :biometric_comparison
```

Comment:
I think it might be a good idea to consider using profile `proofing_components` data to figure out what type of verification it was. An issue I think we could encounter is that technically a user can verify their identity multiple times, and we don't know if an in person enrollment being present necessarily means that their latest profile was proofed that way. It could be from an earlier verification, or they may have cancelled their IPP enrollment and chose to verify online instead (enrollments have statuses, but we're not checking that here either).

I'd suggest using proofing components for IPP and probably GPO (simpler, more performant). Since we already have `User#identity_verified_with_biometric_comparison?`, I think it makes sense to continue using that (dunno that biometric is necessarily exposed through proofing components).

Reply:
Related Slack discussion: https://gsa-tts.slack.com/archives/C03DCUECFH7/p1726594700506499

Maybe it's easiest to just log `proofing_components` as-is. We do that already for plenty of other events, and the IdV detail was considered "extra" for this ticket, I don't think we need to overcomplicate it.

Reply:
I have it logging proofing_components now. It doesn't seem to reveal much to do with id method though.
![Screenshot 2024-09-17 at 3 21 12 PM (2)](https://github.com/user-attachments/assets/9b5718a0-9de5-48e0-a182-849a75b9dfd1)

Reply:
I think it's fine for a first pass.